docs/guidelines/content/subsets.md

```diff
@@ -90,45 +89,42 @@ Restrictive challenges where player units or equivalent must never be used again
+|Collaborative Author of Core Set |• Subset must be approved by [Developer Compliance](https://retroachievements.org/createmessage.php?t=DevCompliance&s=Subset%20Approval%20Request%20-%20[Game%20Name]%20-%20[Subset%20Name]) ²³<br> • Requires a plan and revision vote?<br> &nbsp;&nbsp;&nbsp;◦ No, if all active authors of the set approve<br> &nbsp;&nbsp;&nbsp;◦ Yes, otherwise<br> • Considered a free claim ⁴
+|No Core Set Authorship|• Subset must be approved by [Developer Compliance](https://retroachievements.org/createmessage.php?t=DevCompliance&s=Subset%20Approval%20Request%20-%20[Game%20Name]%20-%20[Subset%20Name]) ²³<br> • If approved, a set plan must be posted in the base set's forum topic and must go through revision voting<br> • Is **not** considered a free claim<br> • In the event that both a revision and subset plan are made for a set, both claims will count as one
+|Junior Developer|• In addition to the above cases, must first be approved by Code Reviewers
 
 ¹ In the case of multiple revision authors, use the most restrictive ruleset in the table. Example: If there is a subset collaboration and any subset author is not a core set author, use the `No Core Set Authorship` rules.
 
 ² To Contact Developer Compliance, send them a site message [here](https://retroachievements.org/createmessage.php?t=DevCompliance&s=Subset%20Approval%20Request%20-%20[Game%20Name]%20-%20[Subset%20Name]) and replace the `[Game Name]` and `[Subset Name]` in the subject line with the game name and desired subset name.
 
-³ _The following are pre-approved subset types and therefore do not require contacting Developer Compliance:_<br>
+³ The following are pre-approved subset types and therefore do not require contacting Developer Compliance:
 
-- Bonus (only if composed of other pre-approved types)<br>
-- Multi<br>
-- DLC/Expansions<br>
-- Extreme or Brutal Challenges<br>
-- Glitch Showcases<br>
-- Regional Differences<br>
-- User-Generated Content<br>
-- Challenge Run: Low Level Game/No Leveling Runs<br>
-- Challenge Run: Solo Class/Monotype Runs<br>
-- Challenge Run: Professor Oak Challenges for official Pokemon main series releases<br>
-- Challenge Run: Perma-Death Challenges, such as Nuzlocke Challenges<br>
+- Bonus (only if composed of other pre-approved types)
+- Multi
+- DLC/Expansions
+- Extreme or Brutal Challenges
+- Glitch Showcases
+- Regional Differences
+- User-Generated Content
+- Challenge Run: Low Level Game/No Leveling Runs
+- Challenge Run: Solo Class/Monotype Runs
+- Challenge Run: Professor Oak Challenges for official Pokemon main series releases
+- Challenge Run: Perma-Death Challenges, such as Nuzlocke Challenges
 
-⁴ _For now, an admin must be contacted via Discord or by [messaging RAdmin](https://retroachievements.org/user/RAdmin) to mark a claim as free. As a courtesy, please only request this if you actually need the claim slot or if you expect a long development period._
 
-### Notes
```

Comment:
This section is under DevCompliance authority and not something that can be removed during a clean up edit.

Reply:
That section was redundant, as ¹ said essentially the same thing but with better formatting (yes really, compare both of them).

Reply:
Sometimes redundancy is intentional to ensure the consumer catches the rule. In any case, please do not edit devcoc content.

Reply:
Before @suspect15 said that, I wasn't aware about that, so I'll revert this and turn it into a draft for now.